zetta_utils/mazepa_layer_processing/alignment/compute_alignment_quality.py

```diff
@@ -0,0 +1,174 @@
+import datetime
+from typing import Any, Dict, List, TypeVar
+
+import torch
+from typeguard import typechecked
+from typing_extensions import ParamSpec
+
+from zetta_utils import builder, log, mazepa
+from zetta_utils.layer import Layer, LayerIndex
+from zetta_utils.layer.volumetric import VolumetricIndex, VolumetricIndexChunker
+from zetta_utils.mazepa import Dependency
+from zetta_utils.typing import Vec3D
+
+logger = log.get_logger("zetta_utils")
+
+IndexT = TypeVar("IndexT", bound=LayerIndex)
+P = ParamSpec("P")
+R_co = TypeVar("R_co", covariant=True)
+
+
+@mazepa.taskable_operation
+def compute_misalignment_stats(**kwargs) -> Dict[str, Any]:
```

Comment:
Can you get rid of the `**kwargs` here?

Reply:
Yeah

Reply:
Wait, why is it better to get rid of kwargs? Is it dangerous? I think this is how your taskable operations were done before.

Reply:
Arbitrary kwargs make code hard to read and hinder static typechecking.

I think the places you're referring to are probably wrapper functions that accept specific kwargs typed by ParamSpec.